core/src/main/java/com/linecorp/armeria/server/http/metric/PrometheusExporterHttpService.java

```diff
@@ -36,23 +37,23 @@
 public class PrometheusExporterHttpService extends AbstractHttpService {
     private static final MediaType CONTENT_TYPE_004 = MediaType.parse(TextFormat.CONTENT_TYPE_004);
 
-    private final CollectorRegistry prometheusRegistry;
+    private final CollectorRegistry collectorRegistry;
 
     /**
      * Create a {@link PrometheusExporterHttpService} instance.
-     * @param prometheusRegistry Prometheus registry
+     * @param collectorRegistry Prometheus registry
      */
-    public PrometheusExporterHttpService(CollectorRegistry prometheusRegistry) {
-        Objects.requireNonNull(prometheusRegistry, "prometheusRegistry");
-        this.prometheusRegistry = prometheusRegistry;
+    public PrometheusExporterHttpService(CollectorRegistry collectorRegistry) {
+        requireNonNull(collectorRegistry, "collectorRegistry");
+        this.collectorRegistry = collectorRegistry;
     }
 
     @Override
     protected void doGet(ServiceRequestContext ctx, HttpRequest req,
                          HttpResponseWriter res) throws Exception {
         ByteArrayOutputStream stream = new ByteArrayOutputStream();
         try (OutputStreamWriter writer = new OutputStreamWriter(stream)) {
-            TextFormat.write004(writer, prometheusRegistry.metricFamilySamples());
+            TextFormat.write004(writer, collectorRegistry.metricFamilySamples());
```

Comment:
@kojilin @jmostella Do I need to sort the samples returned by `metricFamilySamples()` by their names?